doc/classes/TextServerExtension.xml

```diff
@@ -1190,24 +1190,31 @@
 		<method name="_get_support_data" qualifiers="virtual const">
 			<return type="PackedByteArray" />
 			<description>
-				[b]Optional.[/b]
+				[b]Optional, should be implemented if [constant TextServer.FEATURE_USE_SUPPORT_DATA] feature flag is set.[/b]
```

Comment:
What do you mean by "should be implemented"? Like, "mandatory if"?

Reply:
> Like, "mandatory if"?

Yes. I have removed these changes since it's no longer relevant to the PR.